doc/ja/book.xml

```diff
@@ -1350,9 +1350,15 @@ this page has been readed ${page/countRead} times"/>
           <code>==</code> : <code>EQ</code> (等しい)
           </para></listitem>
           <listitem><para>
+          <code>===</code> : <code>EQQ</code> (厳密に等しい)
```

Comment:
No idea how to validate this change 😁

Reply:
Trust me bro, ai and Google gave me the same result :)

Reply:
I don't trust either of those sources :-P